### PR TITLE
feat(quantic): add labels for buttons

### PR DESCRIPTION
**Pull request by MSLI-Coveo**
[SVCC-2299](https://coveord.atlassian.net/browse/SVCC-2299)
feat(quantic): add labels for buttons

After:
Screen Shot 2022-08-23 at 4 15 52 PM
Screen Shot 2022-08-23 at 4 14 38 PM
Screen Shot 2022-08-23 at 4 41 53 PM

### CODE REVIEW

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -70,13 +70,27 @@
         <protected>false</protected>
         <shortDescription>Clear filter</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ClearFilterFacet</fullName>
+        <value>Clear filter for the {{0}} facet.</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>Clear filter for the potato facet.</shortDescription>
```
Comment (erocheleau) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:78]:
```suggestion
        <shortDescription>Clear filter for the filetype facet.</shortDescription>
```

Let's keep the examples cleaner a little bit ;)

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 2]
```diff
@@ -70,13 +70,27 @@
...
+    </labels>
     <labels>
         <fullName>quantic_ClearFilter_plural</fullName>
         <value>Clear {{0}} filters</value>
         <language>en_US</language>
         <protected>false</protected>
         <shortDescription>Clear 2 filters</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ClearFilterFacet_plural</fullName>
```
Comment (erocheleau) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:88]: Could we simplify our life a little bit and just have a "clear all filters for the [value] facet" that always apply?

File: packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <button class={buttonClass} onclick={openQuickview} tooltip={buttonLabel} title={buttonLabel} disabled={hasNoPreview}>
+  <button class={buttonClass} onclick={openQuickview} tooltip={buttonLabel} title={buttonLabel} disabled={hasNoPreview} aria-label={buttonAriaLabelValue}>
```
Comment (erocheleau) [inline packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.html:2]: I like it.

**Review by erocheleau — COMMENTED** [review]
Only thing I want to make sure is if we could simplify this by about half while having a more generic label for the "clear filter" button? 

Just say: "Clear all filters" even if there is only 1 filter active?

**Review by btaillon — APPROVED** [review]
accessibility looks good to me!

**Review by acarpentier01 — APPROVED** [review]
Looks great! Make sure to apply @erocheleau's feedback. 

As a general comment, if the majority of screen readers tend to add punctuation before attributes, then I would remove the period at the end of the descriptions to avoid the "period followed by a comma situation" like in your first screenshot "e.g., "...for preview**.,** button,...". It that's not the case, then keep the periods.

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (acarpentier01) [reply packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:78]: filetype-->file type

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 4]
```diff
@@ -170,10 +184,10 @@
     </labels>
     <labels>
         <fullName>quantic_ShowNResultsPerPage</fullName>
-        <value>Show {{0}} number of results per page</value>
+        <value>Show {{0}} results per page</value>
         <language>en_US</language>
         <protected>false</protected>
-        <shortDescription>Show N results per page.</shortDescription>
+        <shortDescription>Show N results per page</shortDescription>
```
[the remark this answers is not in the document]
Comment (MSLI-Coveo) [reply packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml]: label message changed

**Comment by MSLI-Coveo** [comment]
> Looks great! Make sure to apply @erocheleau's feedback.
> 
> As a general comment, if the majority of screen readers tend to add punctuation before attributes, then I would remove the period at the end of the descriptions to avoid the "period followed by a comma situation" like in your first screenshot "e.g., "...for preview., button,...". It that's not the case, then keep the periods.

Thanks for the suggestions!!

**Review by erocheleau — APPROVED** [review]

**Pull request merged**